keyboards/misonoworks/chocolatebar/readme.md

```diff
@@ -0,0 +1,15 @@
+# Chocolate Bar

```

Comment:
Please use Unix line endings for all files.

Reply:
added!